### PR TITLE
PizDaint (CSCS): Update Profiles

### PR DESCRIPTION
**Pull request by ax3l**
Update to the latest modules and add libSplash for HDF5 support.

I drafted a very specific, quickly outdated, hands on install description in
  https://gist.github.com/ax3l/68cb4caa597df3def9b01640959ea56b
which just summarizes what is already [in the manual](https://picongpu.readthedocs.io/en/latest/install/profile.html#piz-daint-cscs).

ccing @HighIander @BeyondEspresso

### CODE REVIEW

File: etc/picongpu/pizdaint-cscs/picongpu.profile.example [hunk 1]
```diff
@@ -70,8 +81,8 @@ export TBG_TPLFILE="etc/picongpu/pizdaint-cscs/normal.tpl"
 # helper tools ################################################################
 
 # allocate an interactive shell for one hour
-#   getInteractive 2  # allocates to interactive nodes (default: 1)
-getInteractive() {
+#   getNode 2  # allocates to interactive nodes (default: 1)
+getNode() {
```
Comment (ax3l) [inline etc/picongpu/pizdaint-cscs/picongpu.profile.example]: @psychocoderHPC still works for me.

as usual:
```bash
getNode
# wait

# srun commandOnNode
srun hostname
```

File: etc/picongpu/pizdaint-cscs/picongpu.profile.example [hunk 2]
```diff
@@ -26,12 +42,14 @@ module load cray-hdf5-parallel/1.10.0
 #
 # needs to be compiled by the user
 export BOOST_ROOT=$SCRATCH/lib/boost-1.62.0
-export PNGWRITER_ROOT=$SCRATCH/lib/pngwriter
+export PNGWRITER_ROOT=$SCRATCH/lib/pngwriter-0.6.0
```
Comment (PrometheusPi) [inline etc/picongpu/pizdaint-cscs/picongpu.profile.example]: QUESTION:
If this dependency is compiled by the user anyway why bother to add a version number?

Comment (ax3l) [reply etc/picongpu/pizdaint-cscs/picongpu.profile.example]: just a best practice one can do, so one knows at a glance what version is residing in there

File: etc/picongpu/pizdaint-cscs/picongpu.profile.example (hunk 1 above)
Comment (psychocoderHPC) [reply etc/picongpu/pizdaint-cscs/picongpu.profile.example]: yes it is running I not realized that I am using an old profile.

**Pull request merged**